AFP-Medialab/verification-plugin
# hiding credibility signals from non beta_tester users

Checking the synthetic image and deep fake image and video tools are hidden or shown correctly on login and logout:
- not logged in user = hidden
- logged in but not a beta_tester = hidden
- logged in as a beta_tester = shown

Additional changes
- added which domains can see Deepfake Video tool as a recommended tool
- set size of `DownloadIcon` in recommended tools changed for consistency
- fixed issue with Prev Fact Checks where "None detected" and "Please log in..." messages appearing at same time
- Credibility Signals card (Previous Fact Checks and Machine Generated Text) only appears when a `BETA_TESTER` is logged in

Video links for testing:
- YouTube https://www.youtube.com/watch?v=Bu5rps9s5tI
- X https://x.com/BreitbartNews/status/1846249875656065136
- Instagram https://www.instagram.com/wearebreitbart/reel/DDVBcC3uyZE/?hl=en
- Facebook https://www.facebook.com/watch/?v=938175568183623&ref=sharing
- TikTok https://www.tiktok.com/@henneberg.v8/video/7437198470093524246?lang=en
- Telegram https://t.me/WeAreBREITBART/13793
- YouTubeShorts https://www.youtube.com/shorts/KWad5Mer46M
- DailyMotion https://www.dailymotion.com/video/xvmg1a
- LiveLeak https://www.liveleak.com/view?t=CoQQl_1607822811
- Vimeo https://vimeo.com/362710097
- Mastodon https://mastodon.social/@radioklara/113622400445884803
- VK https://vk.com/wall-57424472_432185
